### ISSUE TITLE
Async Issue with transform

### ISSUE THREAD

**Issue by Tirumala-K** [WorkMaze/JUST.net#260]
We are facing an issue of jumbled transformed values when we are processing our data through async programming using Transformer. Do you have any transform function which supports async programming?

**Comment by Courela** [comment]
No, currently there are no async methods. Current version uses instances, so it should work with async programming. Older versions were static and were not thread safe.

**Comment by Tirumala-K** [comment]
Currently we are using 4.2.0 version. Is the version thread safe? 
![image](https://user-images.githubusercontent.com/26627790/214321299-a5251246-ff6a-4a59-be71-3d2f70b5b43c.png)

**Comment by Courela** [comment]
Yes, it should be thread safe.
